src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseHybridFVMKernels.hpp

```diff
@@ -290,16 +287,16 @@ class ElementBasedAssemblyKernel
     globalIndex upwDofNumber = 0;
 
     // for each element, loop over the one-sided faces
-    for( integer ifaceLoc = 0; ifaceLoc < NUM_FACE; ++ifaceLoc )
+    for( integer iFaceLoc = 0; iFaceLoc < NUM_FACE; ++iFaceLoc )
     {
 
       // 1) Find if there is a neighbor, and if there is, grab the indices of the neighbor element
       localIndex local[3] = { m_er, m_esr, ei };
       localIndex neighbor[3] = { m_er, m_esr, ei };
-      bool const hasNeighbor =
+      bool const isNeighborFound =
```

Comment:
My comment about the `isNeighborFound` was more to say that the function `findNeighbor` is not properly named imho. 😉

Reply:
Ah ok, sorry I misunderstood, I will rename it

Reply:
don't worry, `isNeighborFound` is nice too.

Reply:
Done, I renamed the function as well